packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -1,229 +1,412 @@
-.aa-Form {
-  flex-grow: 1;
-  margin: 0;
-  position: relative;
+// ----------------
+// Variables
+// ----------------
+:root {
```

Comment:
this should probably be set on `.aa-Form` (or whatever else is the wrapper)

Reply:
I've added prefix to the variables so it should be ok to use them on roots.